app/app/settings.py

```diff
@@ -238,12 +240,142 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 STATICFILES_STORAGE = env('STATICFILES_STORAGE', default='app.static_storage.SilentFileStorage')
+
 STATICFILES_DIRS = env.tuple('STATICFILES_DIRS', default=('assets/', ))
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'pipeline.finders.PipelineFinder',
+)
 STATIC_ROOT = root('static')
 
 STATIC_HOST = env('STATIC_HOST', default='')
 STATIC_URL = STATIC_HOST + env('STATIC_URL', default='/static/')
 
+PIPELINE = {
+    'PIPELINE_ENABLED': True,
+    'PIPELINE_COLLECTOR_ENABLED': True,
+    'COMPILERS': ('pipeline.compilers.es6.ES6Compiler', ),
```

Comment:
Would the compiler run against all js files or only the ones mentioned in the Pipeline ?
Should we export this to a new file (cause this is gonna keep growing in size)

Reply:
🙌I was wondering if I should split it off... Yeah, we can move this into it's own file and import it in settings.py 👍